### ISSUE TITLE
New rules: if conditions should not be too long

### ISSUE THREAD

**Issue by akuleshov7** [cqfn/diKTat#893]
We need to add a rule to control how long are if conditions

**Comment by kgevorkyan** [comment]
Duplicate of https://github.com/cqfn/diKTat/issues/829 ?

**Comment by petertrr** [comment]
No, this issue is about lines of code inside branches, while #829 is about conditions. @akuleshov7, I think we should rename this issue to be not so confusing
